### PR TITLE
Add a jump button to investigate logs

### PR DESCRIPTION
**Pull request by Tourte-Yaya**
## What Does This PR Do

Add a jump button to investigate logs in the investigate verbs
## Why It's Good For The Game

Quality of life for the admins while investigating is good.

## Changelog
:cl:
add: Added a jump button in investigate logs.
/:cl:

### CODE REVIEW

File: code/modules/admin/admin_investigate.dm [hunk 1]
```diff
@@ -20,7 +20,7 @@
 	var/F = investigate_subject2file(subject)
 	if(!F)	return
 	GLOB.investigate_log_subjects |= subject
-	F << "<small>[time_stamp()] \ref[src] ([x],[y],[z])</small> || [src] [message]<br>"
+	F << "<small>[time_stamp()] \ref[src] ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)</small> || [src] [message]<br>"
```
Comment (hal9000PR) [inline code/modules/admin/admin_investigate.dm:23]: do admins ever make use of the \ref?

**Review by hal9000PR — APPROVED** [review]
code is fine.

File: code/modules/admin/admin_investigate.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/modules/admin/admin_investigate.dm:23]: I do

Comment (AffectedArc07) [inline code/modules/admin/admin_investigate.dm:23]:
```suggestion
	F << "<small>[time_stamp()] \ref[src] [ADMIN_COORDJMP(src)] </small> || [src] [message]<br>"
```
Cleaner. Try this.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**